src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# ListBoxOption:
+
+Provides an ListBoxOption component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (node array; required)
+`children` specifies the array of elements that will be displayed within the ListBoxOption component.
+
+### `className` (string; optional)
+`className` specifies Bootstrap class names to apply to the ListBoxOption component. The default is an empty string.
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered ListBoxOption component. The default is div. Example alternatives include li, Button, etc.
```

Comment:
Is there a reason not to use `type` instead of `tag`? I get that it's referencing `HTML` tags, but `Button`, for example, is not a valid `HTML` tag (it is a valid tag for a component in `JSX`)

Reply:
I can change it to type. Just so that I'm sure I understand correctly, you're not advocating for supporting anything but HTML tags here, right?

Reply:
Oh I misunderstood this - when I saw `Button` I thought that meant a `Button` component.

If that's not the case, and it's only HTML tags, then I think `tag` is fine!

Reply:
Also, for the `PropType` you could use `oneOf` https://github.com/sindresorhus/html-tags

Reply:
Ack! Sorry, that's my mistake. I did actually mean Button, but a string "Button" is pretty useless. Hm. Is there even a way we could support a JSX component in this way?

Reply:
Check [this StackBlitz example](https://stackblitz.com/edit/dynamically-create-react-component) out

```jsx
import React, { Component } from 'react';
import { render } from 'react-dom';
import Foo from './Foo';
import Bar from './Bar';
import './style.css';

class App extends Component {
  constructor() {
    super();
    this.state = {
      name: 'React'
    };
  }

  render() {
    return (
      <div>
        <Bar component={Foo} />
      </div>
    );
  }
}

render(<App />, document.getElementById('root'));
```